docs/source/getting_started/index.rst

```diff
@@ -0,0 +1,12 @@
+Getting started
+===============
+
+
+.. include:: install.rst
+
+.. toctree::
+   :maxdepth: 2
+
+    Quick-start notebook <example_notebooks/dowhy_simple_example>
```

Comment:
The quick-start notebook and comparison are not visible on https://petergtz.github.io/dowhy/master/getting_started/index.html. Where will these be shown?

Reply:
That's an error. Will investigate.

Reply:
Okay, I've restructured this completely. Please have a look.